game/scripts/npc/items/item_bfury.txt

```diff
@@ -60,7 +60,7 @@
       "02"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "bonus_health_regen"                              "7.5 11 14 17 20"
+        "bonus_health_regen"                              "0" //OAA
```

Comment:
y tho

Reply:
hp regen does nothing on this item, bfury is too expensive imo, giving melee heroes some love. likely imba but better direction imo

Reply:
We can argue that regen is useless on some heroes, but I offer a better solution above and probably more imba solution.